[2025-01-02T15:47:32.464+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-01-02T15:47:32.660+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: api_to_kafka_pipeline.fetch_and_produce_data scheduled__2024-12-27T16:00:00+00:00 [queued]>
[2025-01-02T15:47:32.752+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: api_to_kafka_pipeline.fetch_and_produce_data scheduled__2024-12-27T16:00:00+00:00 [queued]>
[2025-01-02T15:47:32.754+0000] {taskinstance.py:2866} INFO - Starting attempt 4 of 3
[2025-01-02T15:47:32.962+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): fetch_and_produce_data> on 2024-12-27 16:00:00+00:00
[2025-01-02T15:47:32.975+0000] {logging_mixin.py:190} WARNING - /home/airflow/.local/lib/python3.12/site-packages/airflow/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=127) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-01-02T15:47:33.050+0000] {standard_task_runner.py:72} INFO - Started process 161 to run task
[2025-01-02T15:47:33.052+0000] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'api_to_kafka_pipeline', 'fetch_and_produce_data', 'scheduled__2024-12-27T16:00:00+00:00', '--job-id', '179', '--raw', '--subdir', 'DAGS_FOLDER/api_to_kafka_dag.py', '--cfg-path', '/tmp/tmppsltiizp']
[2025-01-02T15:47:33.055+0000] {standard_task_runner.py:105} INFO - Job 179: Subtask fetch_and_produce_data
[2025-01-02T15:47:33.274+0000] {task_command.py:467} INFO - Running <TaskInstance: api_to_kafka_pipeline.fetch_and_produce_data scheduled__2024-12-27T16:00:00+00:00 [running]> on host d83e04c44ca5
[2025-01-02T15:47:33.755+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='api_to_kafka_pipeline' AIRFLOW_CTX_TASK_ID='fetch_and_produce_data' AIRFLOW_CTX_EXECUTION_DATE='2024-12-27T16:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='4' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-12-27T16:00:00+00:00'
[2025-01-02T15:47:33.757+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-01-02T15:47:33.856+0000] {api_to_kafka_dag.py:85} INFO - Fetching data for IBM at 5min interval
[2025-01-02T15:47:34.232+0000] {api_to_kafka_dag.py:93} INFO - Processing and sending data to Kafka
[2025-01-02T16:14:18.486+0000] {local_task_job_runner.py:127} ERROR - Received SIGTERM. Terminating subprocesses
[2025-01-02T16:14:18.685+0000] {process_utils.py:132} INFO - Sending 15 to group 161. PIDs of all processes in the group: [161]
[2025-01-02T16:14:18.690+0000] {process_utils.py:87} INFO - Sending the signal 15 to group 161
[2025-01-02T16:14:19.085+0000] {local_task_job_runner.py:127} ERROR - Received SIGTERM. Terminating subprocesses
